modules/cnv-uploading-local-disk-image-dv.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-uploading-local-disk-images-block.adoc
+
+[id="cnv-uploading-local-disk-image-dv_{context}"]
+= Uploading a local disk image to a new DataVolume
+
+You can use the `virtctl` CLI utility to upload a virtual machine disk image from
+a client machine to a DataVolume (DV) in your cluster. Once uploaded, the DV can be 
+added to a virtual machine.
+
+.Prerequisites
+
+* A virtual machine disk image, in RAW, ISO, or QCOW2 format, optionally
+compressed by using `xz` or `gz`.
+* The `kubevirt-virtctl` package must be installed on the client machine.
+* The client machine must be configured to trust the {product-title} router's
+certificate.
+* A spare DataVolume the same size or larger than the disk you are uploading.
+
+.Procedure
+
+. Identify the following items:
+* File location of the VM disk image you want to upload
+* Name of the DataVolume
+
+. In the correct namespace, run the `virtctl image-upload` command to upload your disk image.
```

Comment:
What's the correct namespace?

Reply:
Have dropped that so that it is the same as the `uploading-local-disk-image-pvc` procedure.